static/js/services/select2-search.js

```diff
@@ -88,17 +89,44 @@ angular.module('inboxServices').factory('Select2Search',
           });
       };
 
-      var resolveInitialValue = function(selectEl) {
+      var resolveInitialValue = function(selectEl, initialValues) {
+        // TODO: add support for multiples values here
+        initialValues = initialValues[0];
+
+        if (initialValues) {
+          if (!selectEl.children('option[value="' + initialValues + '"]').length) {
+            selectEl.append($('<option value="' + initialValues + '"/>'));
```

Comment:
When we have the ID and the display name for the initial value shouldn't the content of this option be set to the display name? Or does select2 magically sort it out?

I suspect the `realTo` code above strips out the name so all we have is the ID which will render instead of the user friendly name. This would be a regression from the 0.4 codebase.